### DISCUSSION TITLE
How to store ROOK logs persistently?What do you recommend?

### DISCUSSION THREAD

**Discussion by jackcongx** [rook/rook#14272] [Q&A] [answered]
How to store ROOK logs persistently?What do you recommend?

**Comment by travisn** [comment, marked as answer]
Which logs do you want to store? All of the pods including the rook operator and ceph daemon pods?

**Reply by jackcongx** [reply]
yes,All of the pods including the rook operator and ceph daemon pods

**Reply by jackcongx** [reply]
@travisn

**Reply by travisn** [reply]
@jackcongx Have you looked into the Elasticsearch stack?
